cmd/ttn-lw-stack/commands/start.go

```diff
@@ -170,6 +170,20 @@ var startCommand = &cobra.Command{
 
 		redisConsumerID := redis.Key(host, strconv.Itoa(os.Getpid()))
 
+		transport, err := c.HTTPTransport(ctx)
+		if err != nil {
+			return err
+		}
+		if cfg := config.ServiceBase.DeviceRepository; cfg.Transport == nil {
```

Comment:
```suggestion
		if conf := config.ServiceBase.DeviceRepository; conf.Transport == nil {
```
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.10/DEVELOPMENT.md#single-word-entities